lib/tabula.ex

```diff
@@ -61,6 +62,11 @@ defmodule Tabula do
     def keys(row), do: row |> Keyword.keys
   end
 
+  defimpl Row, for: Any do
+    def get(%{__struct__: _} = row, col, default \\ nil), do: row |> Map.get(col, default)
+    def keys(row), do: row |> Map.keys |> tl
```

Comment:
Once again you're right @aerosol...😞

I've modified the code as you suggested and added a few test about the `Table.Row` protocol implementations but sincerely I'm not sure about the style I've used.

If you want I can separate each test (i.e. one test per assertion) and add a better description to each one of them.